src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -1,357 +1,374 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Same thing. Why does GitHub think that the files are that different? spaces/tab issues?

Reply:
Same - It's due to git merge from master branch.